assets/js/components/PresenterStreams.tsx

```diff
@@ -46,26 +48,26 @@ const PresenterStreams = ({
     syncPresenters(eventChannel, setPresenters);
   }, [eventChannel]);
 
-  return presenters.includes(clientName) ? (
+  return presenters.some((e) => e.name === client.name) ? (
     <div className={`PresenterStreams ${mode == "hls" ? "Hidden" : ""}`}>
       <div className={`StreamsGrid Grid${presenters.length}`}>
         {presenters.map((presenter) => {
           return (
             <RtcPlayer
-              isMyself={clientName == presenter}
-              name={presenter}
+              isMyself={client == presenter}
+              presenter={presenter}
               playerCallbacks={playerCallbacks}
-              key={presenter}
+              key={presenter.name}
```

Comment:
Where do we use this `key`?

Reply:
because you have to pass it when you are listing components